pkgs/top-level/all-packages.nix

```diff
@@ -15795,7 +15795,7 @@ with pkgs;
 
   webcamoid = libsForQt5.callPackage ../applications/video/webcamoid { };
 
-  webcord = callPackage ../by-name/we/webcord/package.nix { electron = electron_32; };
```

Comment:
As per https://github.com/NixOS/nixpkgs/pull/370758#issuecomment-2592682435, this should have been
```
{ electron = electron_33; };
```